gradle/gradle-mvn-push.gradle

```diff
@@ -83,6 +83,13 @@ afterEvaluate { project ->
         source = android.sourceSets.main.java.srcDirs
         exclude "**/internal/**"
         classpath += project.files(android.getBootClasspath().join(File.pathSeparator))
+
+        // Append also the classpath and files for release library variants.
+        // This fixes the javadoc warnings.
+        // Copy pasta from https://github.com/novoda/bintray-release/pull/39/files
+        def releaseVariant = project.android.libraryVariants.toList().last()
```

Comment:
Using `.last()` seems brittle.  How about:
```
      android.libraryVariants.all { variant ->
          if (variant.equals("release") {
              classpath += variant.javaCompile.classpath
              classpath += variant.javaCompile.outputs.files
          }
      }
```

Reply:
I'll try!

Reply:
Then that if statement needs to be a bit more sophisticated.

Reply:
s/equals/endsWith/